examples/example_fixture.h

```diff
@@ -91,34 +91,34 @@ struct Benchmark : public ::testing::Test {
   void SetUp() {
     if (!FLAGS_disable_reproducibility_checks) {
```

Comment:
sorry for iteration, can we also rename this flag?  (and change autotuning commands stored somewhere in the code base)?